pkgs/servers/tigerbeetle/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig
+}:
+stdenv.mkDerivation rec {
+  pname = "tigerbeetle";
+  version = "unstable-2022-09-30";
```

Comment:
The version does not match what you actually use in `rev =` below. I think it should be `2023-02-13-weekly`. No?

Reply:
I thought since these versions are pre-release it makes more sense to use `unstable`. However, I think now IIRC `unstable-DDDD-MM-YY` is only for packages without tags.